usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -11,7 +11,10 @@ is_true "$GRUB_RESCUE" || return
 # (grub-probe or grub2-probe only exist in GRUB 2)
 # in particular do not run this script when GRUB Legacy is used
 # (for GRUB Legacy output/default/94_grub_rescue.sh is run):
-type -p grub-probe >&2 || type -p grub2-probe >&2 || { LogPrint "Skipping GRUB_RESCUE setup for GRUB 2 (no GRUB 2 found)." ; return ; }
+if [[ ! $( type -p grub-probe ) && ! $( type -p grub2-probe ) ]]; then
```

Comment:
I liked to have the 'type -p' stdout output in the log file.
I thought having a verbose log file is good in general.
In contrast by default rear should be silent
on the terminal where rear was launched
cf. https://github.com/rear/rear/issues/929
Therefore I redirected stdout to stderr because
stderr gets logged but not shown on the terminal
cf. https://github.com/rear/rear/issues/885